pulsar-zookeeper-utils/src/main/java/org/apache/pulsar/zookeeper/ZkIsolatedBookieEnsemblePlacementPolicy.java

```diff
@@ -142,11 +141,6 @@ private ZooKeeperCache getAndSetZkCache(Configuration conf) {
     public PlacementResult<List<BookieId>> newEnsemble(int ensembleSize, int writeQuorumSize, int ackQuorumSize,
             Map<String, byte[]> customMetadata, Set<BookieId> excludeBookies)
             throws BKNotEnoughBookiesException {
-        // parse the ensemble placement policy from the custom metadata, if it present, we will apply it to
```

Comment:
so this is the fix.
can you please remove all unnecessary code formatting ?

Reply:
I am sorry. Could you please tell me more? It seems that the line you comment already outdated.